contrib/opencensus-exporter-ocagent-trace/README.rst

```diff
@@ -0,0 +1,16 @@
+OpenCensus OC-Agent Trace Exporter
+============================================================================
+
+Installation
+------------
+
+::
+
+    pip install opencensus-exporter-ocagent-trace
```

Comment:
@c24t I wish to get your feedback on the package name and namespace.

Reply:
I chose `opencensus-exporter-xyz` instead of `opencensus-ext` to make it easier for people to tell exporters from integrations.
I chose namespace like `opencensus.ext.ocagent.trace_exporter`, so it is possible for the trace exporter and stats/log exporter to share the same namespace `opencensus.ext.ocagent`.
This also makes it possible to have a single package `opencensus-exporter-ocagent` which provides multiple exporters in the future.

Reply:
@tjzo I'd like to hear your feedback as well.

Reply:
I think the namespace opencensus.ext.ocagent. would be fine.
And I would like to name the package opencensus-ext-exporter-ocagent. Keep all the package name starts with the same prefix.
The go and ruby package name is opencensus-go-exporter-ocagent / opencensus-ruby-exporter-ocagent, but they are all in their own repo. If we just place them in this repo, I think opencensus-ext prefix is fine.

Reply:
@tjzo Thanks for the quick reply! I'm fine with either `opencensus-ext-exporter-ocagent` or `opencensus-exporter-ocagent`.

@c24t Looks like OpenCensus Go does support "one exporter" for both trace and metrics/stats:
https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/blob/060e6898ab87d943e2ff1090efb927fbc5181cea/ocagent.go#L103

Reply:
The distribution name, package name, and directory structure all look good to me, but it's a little odd that exporters don't have `ext` in the distribution name but do have it in the namespace/package name. Do you think it's more important to distinguish between exporter and integration distributions than packages?

What about using the `opencensus.exp` namespace for exporter code and using the prefix `opencensus-exp` for exporter distributions? E.g. `opencensus-exp-ocagent` This makes the distribution and package names more consistent (nice to have, but not necessarily a goal) and the distribution names shorter.

This is largely a matter of taste, and I'm fine with the names you proposed. Just a preference for shorter names.

Reply:
@tjzo @c24t probably we can just try not to distinguish between exporters and integrations.
How about just calling it `opencensus-ext-ocagent`?

I tend to avoid `exp` since we will have to use something like `prop` or `pro` for propagators if we decided to extract them.

Reply:
I'll merge the PR for now and continue working on extracting other exporters.
Please let me know your feedbacks and we can adjust the package name anytime before we release.